graphene_sqlalchemy/registry.py

```diff
@@ -1,22 +1,83 @@
+from collections import OrderedDict
+
+from sqlalchemy.types import Enum as SQLAlchemyEnumType
+
+from graphene import Enum
+
+from .utils import to_enum_value_name, to_type_name
+
+
 class Registry(object):
-    def __init__(self):
+    def __init__(self, check_duplicate_registration=False):
```

Comment:
The `get_type_for_model()` method (used by `convert_sqlalchemy_relationship()`) assumes that there is only one graphene type per model. You may want to make sure that this assumption is met and you haven't accidentally associated two different types with one model.

Reply:
Ah, ok - good question. Not checking was the behavior until now, and `test_types` would break because CustomCharacter uses the same model as Character. It only matters if you have relationships. Suggestion: Maybe we should always allow duplicate registration, but memorize all of them instead of only the last. And only when get_type_for_model is called, we throw an error which lists the duplicates. I.e. you only get an error, when this is really a problem.